docs/kubernetes/operator/instrumenting-nodejs.md

```diff
@@ -0,0 +1,163 @@
+This document focuses on instrumenting Node.js applications on Kubernetes, using the OpenTelemetry Operator, the Elastic Distribution of OpenTelemetry (EDOT) Collectors, and the [EDOT Node.js](https://github.com/elastic/elastic-otel-nodejs) SDK.
+
+- For general knowledge about the EDOT Node.js SDK, refer to the [getting started guide](https://github.com/elastic/elastic-otel-node/blob/main/packages/opentelemetry-node/docs/get-started.md) and [configuration](https://github.com/elastic/elastic-otel-node/blob/main/packages/opentelemetry-node/docs/configure.md).
+- For Node.js auto-instrumentation specifics, refer to [OpenTelemetry Operator Node.js auto-instrumentation](https://opentelemetry.io/docs/kubernetes/operator/automatic/#nodejs).
+- For general information about instrumenting applications on kubernetes, refer to [instrumenting applications](./instrumenting-applications.md).
+
+## Instrument a Node.js app with EDOT Node.js SDK on Kubernetes
+
+<!--
+Useful links:
+- Example: https://github.com/elastic/elastic-otel-node/tree/main/examples/otel-operator/ documented at https://github.com/elastic/elastic-otel-node/blob/main/DEVELOPMENT.md#testing-k8s-auto-instrumentation-with-otel-operator
+(not user friendly, but we could use it in the future if we want to add a proper example here)
+-->
+
+In this example, you'll learn how to:
+
+- Enable auto-instrumentation of a Node.js application using one of the following supported methods:
+  - Adding an annotation to the deployment Pods.
+  - Adding an annotation to the namespace.
+- Verify that auto-instrumentation libraries are injected and configured correctly.
+- Confirm data is flowing to **Kibana Observability**.
+
+For this example, we assume the application you're instrumenting is a deployment named `nodejs-app` running in the `nodejs-ns` namespace.
+
+1. Ensure you have successfully [installed the OpenTelemetry Operator](./README.md), and confirm that the following `Instrumentation` object exists in the system:
+
+    ```bash
+    $ kubectl get instrumentation -n opentelemetry-operator-system
+    NAME                      AGE    ENDPOINT
+    elastic-instrumentation   107s   http://opentelemetry-kube-stack-daemon-collector.opentelemetry-operator-system.svc.cluster.local:4318
+    ```
+    > [!NOTE]
+    > If your `Instrumentation` object has a different name or is created in a different namespace, you will have to adapt the annotation value in the next step.
+
+2. Enable auto-instrumentation of your Node.js application using one of the following methods:
+
+    - Edit your application workload definition and include the annotation under `spec.template.metadata.annotations`:
+
+    ```yaml
+    kind: Deployment
+    metadata:
+      name: nodejs-app
+      namespace: nodejs-ns
+    spec:
+    ...
+      template:
+        metadata:
+    ...
+          annotations:
+            instrumentation.opentelemetry.io/inject-nodejs: opentelemetry-operator-system/elastic-instrumentation
+    ...
+    ```
```

Comment:
indentation nit: If you indent this section one more level, then it will be "under" the `- Edit your ...` bullet.  Similarly for some code blocks below.

Reply:
updated, thanks a lot!